packages/services/src/providers/perps-provider.factory.ts

```diff
@@ -1,17 +1,18 @@
-import { config } from "@/config/index.js";
-import { serviceLogger } from "@/lib/logger.js";
-import { SymphonyPerpsProvider } from "@/services/providers/perps/symphony-perps.provider.js";
-import { IPerpsDataProvider, PerpsProviderConfig } from "@/types/perps.js";
+import pino from "pino";
+
+import { IPerpsDataProvider, PerpsProviderConfig } from "../types/perps.js";
+import { SymphonyPerpsProvider } from "./perps/symphony-perps.provider.js";
 
 /**
  * Factory for creating perps data providers based on configuration
  */
 export class PerpsProviderFactory {
+  static logger = pino.default();
   /**
    * Create a perps data provider based on the configuration
    */
   static createProvider(config: PerpsProviderConfig): IPerpsDataProvider {
```

Comment:
nitpick: the logger could be passed as an arg instead of statically initialized to the default.  Not sure if we configure out process-wide logger instances with anything other than the default, but if we did that would ensure that we don't lose that customization in the instances created by this factory